android/src/main/java/io/ably/flutter/plugin/AblyMessageCodec.java

```diff
@@ -136,6 +136,20 @@ private void WriteJsonElement(ByteArrayOutputStream stream, JsonElement value){
         }
     }
 
+    /**
+     * Converts Map to JsonObject, ArrayList to JsonArray and
+     * returns null if these 2 types are a no-match
+     * */
+    static JsonElement readValueAsJsonElement(final Object object) {
+        final Gson gson = new Gson();
+        if(object instanceof Map) {
+            return gson.fromJson(gson.toJson(object, Map.class), JsonObject.class);
+        } else if (object instanceof ArrayList){
```

Comment:
Please include a space before opening curly braces. i.e.:

```suggestion
        } else if (object instanceof ArrayList) {
```

Reply:
Same as https://github.com/ably/ably-flutter/pull/23#discussion_r471921091

052467d338e6fd8a37e3474c03b355e0ec1b8fdc